sql/moz-fx-data-shared-prod/telemetry_derived/rolling_cohorts_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  client_id,
+  first_seen_date AS cohort_date,
+  ANY_VALUE(activity_segment) AS activity_segment,
```

Comment:
Might there be any case where there is more than one value for any of these and only the first is taken?

Reply:
Yeah that might be the case sometimes, in most cases we just take an arbitrary value with nulls being last which is what `any_value` does anyway